assemble/bin/accumulo-cluster

```diff
@@ -405,7 +410,7 @@ function control_services() {
   local group
   local tserver
   local G
-  if [[ $ARG_ALL == 1 && $operation == "stop" ]]; then
+  if [[ $ARG_ALL == 1 && $ARG_LOCAL == 0 && $operation == "stop" ]]; then
```

Comment:
What is the net effect of this change that is the basis for changing it?
I didn't modify this line from what's in the main branch, so I'm not sure the original intent.

Reply:
The original intent of this line is to try and stop all services in the accumulo cluster gracefully. 
We do want to do this when the operation is `stop` and the `ARG_ALL`  option is set. 

However, `stop-here` also meets those two criteria while also setting `ARG_LOCAL`.

This is problematic as a user could go to a node that's running only compactors or scan servers and running `./accumulo-cluster stop-here` expecting the local services would be stopped. 
Instead of only stopping the local services, the script will first attempt to stop all services in the cluster. 

The change to this line requires `ARG_LOCAL` to be unset which restricts the `admin stopAll` command from running  when `stop-here` is run.

Reply:
`ARG_ALL=1` is set on line 174 if no other options for a given command are specified.

```
  # Handle the case where no args were passed
  if [[ $ARG_MANAGER == 0 && $ARG_GC == 0 &&
    $ARG_MONITOR == 0 && $ARG_COORDINATOR == 0 && $ARG_TSERVER == 0 &&
    $ARG_SSERVER == 0 && $ARG_COMPACTOR == 0 ]]; then
    ARG_ALL=1
  fi
  ```